### PR TITLE
Added sinks for flushing producer and committing offsets.

### PR DESCRIPTION
**Pull request by Kheldar**

### CODE REVIEW

**Comment by newhoggy** [comment]
👍

File: src/Kafka/Conduit/Sink.hs [hunk 1]
```diff
@@ -86,3 +88,14 @@ kafkaSink props =
         Just msg -> do
           res <- produceMessage prod msg
           runHandler $ maybe (Right prod) Left res
+
+commitOffsetsSink :: MonadIO m => KafkaConsumer -> Sink i m ()
+commitOffsetsSink = awaitForever . const . commitAllOffsets OffsetCommit
+
+flushProducerSink :: MonadIO m => KafkaProducer -> Sink i m ()
```
Comment (AlexeyRaga) [inline src/Kafka/Conduit/Sink.hs]: I am not sure if this one is actually needed, especially as a Sink

Comment (Kheldar) [inline src/Kafka/Conduit/Sink.hs]: It probably can't hurt.

Comment (AlexeyRaga) [inline src/Kafka/Conduit/Sink.hs]: It probably can't, but I don't think it should be doing it as a part of a library ;)

Comment (Kheldar) [inline src/Kafka/Conduit/Sink.hs]: Why?

**Pull request merged**